google-client-loader.html

```diff
@@ -91,79 +34,85 @@
     </script>
 
 @element google-api-loader
-@extends google-client-api
 @homepage https://googlewebcomponents.github.io/google-apis
 -->
-<polymer-element name="google-api-loader"
-                 attributes="name version appId root"
-                 extends="google-client-api"
-                 on-api-load="{{doneLoadingClient}}">
+<dom-module id='google-client-loader'>
+  <template>
+    <google-js-api on-js-api-load="loadClient"></google-js-api>
+  </template>
+</dom-module>
+
 <script>
   (function() {
     'use strict';
 
     // Stores whether the API client is done loading.
-    var clientLoaded_ = false;
+    var _clientLoaded = false;
 
     // Loaders and loading statuses for all APIs, indexed by API name.
     // This helps prevent multiple loading requests being fired at the same time
     // by multiple google-api-loader elements.
-    var statuses_ = {};
-    var loaders_ = {};
+    var _statuses = {};
+    var _loaders = {};
+
+    Polymer({
+
+      is: 'google-client-loader',
 
-    Polymer('google-api-loader', {
       /**
        * Fired when the requested API is loaded. Override this name
-       * by setting `successEventName`. 
+       * by setting `successEventName`.
        * @event google-api-load
        */
 
       /**
        * Fired if an error occurs while loading the requested API. Override this name
-       * by setting `errorEventName`. 
+       * by setting `errorEventName`.
        * @event google-api-load-error
        */
 
-      /**
-       * Name of the API to load, e.g. 'urlshortener'.
-       *
-       * You can find the full list of APIs on the
-       * <a href="https://developers.google.com/apis-explorer"> Google APIs
-       * Explorer</a>.
-       * @attribute name
-       * @type string
-       * @required
-       */
-      name: '',
-
-      /**
-       * Version of the API to load, e.g. 'v1'.
-       * @attribute version
-       * @type string
-       * @required
-       */
-      version: '',
-
-      /**
-       * App Engine application ID for loading a Google Cloud Endpoints API.
-       * @attribute appId
-       * @type string
-       */
-      appId: null,
-
-      /**
-       * Root URL where to load the API from, e.g. 'http://host/apis'.
-       * For App Engine dev server this would be something like:
-       * 'http://localhost:8080/_ah/api'.
-       * Overrides 'appId' if both are specified.
-       * @attribute root
-       * @type string
-       */
-      root: null,
+      properties: {
+        /**
+         * Name of the API to load, e.g. 'urlshortener'.
+         *
+         * You can find the full list of APIs on the
+         * <a href="https://developers.google.com/apis-explorer"> Google APIs
+         * Explorer</a>.
+         * @attribute name
+         * @type string
+         * @required
+         */
+        name: String,
+
+        /**
+         * Version of the API to load, e.g. 'v1'.
+         * @attribute version
+         * @type string
```

Comment:
Yea, we should probably just remove these then. But I'm ok with followups for doc cleanup